cmd/agent/app/flags.go

```diff
@@ -48,11 +48,10 @@ func (b *Builder) Bind(flags *flag.FlagSet) {
 			p.Server.HostPort,
 			"host:port for the UDP server")
 	}
-	flags.StringVar(
-		&b.CollectorHostPort,
+	flags.Var(
+		&b.CollectorHostPorts,
```

Comment:
rather than adding `staticCollectors` type in config, I would define a type here, e.g. stringSliceFlag, which would take a pointer to a regular slice, e.g.

```go
type stringSliceFlag type {
    slice *[]string
}

flags.Var(
    &stringSliceFlag{slice: &b.CollectorHostPorts},
```

This way the "flag"-ness is encapsulated in the `flags` file, without spilling into the config struct.